Mathlib/Tactic/Linarith/Frontend.lean

```diff
@@ -453,16 +453,18 @@ def elabLinarithArg (tactic : Name) (t : Term) : TacticM Expr := Term.withoutErr
     throwErrorAt t "Argument passed to {tactic} has metavariables:{indentD e}"
   return e
 
+open Mathlib.Tactic.Linarith
```

Comment:
Should `elabLinarithArg` (three lines above) also be namespaced?

Reply:
I think it was a mistake that it was in the global namespace.

It seems to me that `Mathlib.Tactic.Linarith` would be nicer for everything than `Mathlib.Tactic`, but I don't want to think about how to disambiguate the `linarith` parser descriptor and the `linarith` function. This PR is doing what it says it's doing (keeping declarations out of the global namespace) so let's merge.

bors r+